packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -767,6 +767,16 @@ const CommentBoard = ( {
 			? actions.filter( ( item ) => item.isEligible( thread ) )
 			: [];
 
+	let deleteConfirmMessage = __(
+		'Are you sure you want to delete this note?'
+	);
+
+	// When deleting a top level note, descendants will also be deleted.
+	if ( thread.parent === 0 ) {
+		deleteConfirmMessage +=
+			' ' + __( "This will also delete all of this note's replies." );
+	}
```

Comment:
```suggestion
	const deleteConfirmMessage =
		// When deleting a top level note, descendants will also be deleted.
		thread.parent === 0
			? __(
					"Are you sure you want to delete this note? This will also delete all of this note's replies."
			  )
			: __( 'Are you sure you want to delete this reply?' );
```

In all locales, a half-width space does not necessarily indicate a sentence separator. Although it is redundant, I think it is better to do it this way.

Additionally, when deleting a reply, 'Are you sure you want to delete this **reply**?' may be more appropriate.

Reply:
Good point.